src/app/components/VisuallyHiddenText/README.md

```diff
@@ -0,0 +1,55 @@
+# visually-hidden-text - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/simorgh/badge.svg?targetFile=packages%2Fcomponents%2Fsimorgh-visually-hidden-text%2Fpackage.json)](https://snyk.io/test/github/bbc/simorgh?targetFile=packages%2Fcomponents%2Fsimorgh-visually-hidden-text%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/simorgh.svg?path=packages/components/simorgh-visually-hidden-text)](https://david-dm.org/bbc/simorgh?path=packages/components/simorgh-visually-hidden-text) [![peerDependencies Status](https://david-dm.org/bbc/simorgh/peer-status.svg?path=packages/components/simorgh-visually-hidden-text)](https://david-dm.org/bbc/simorgh?path=packages/components/simorgh-visually-hidden-text&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/simorgh/?path=/story/visuallyhiddentext--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/simorgh/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/#legacy/simorgh-visually-hidden-text/src.svg)](https://www.npmjs.com/package/#legacy/simorgh-visually-hidden-text/src) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+Adds non-visible text to the page, to be utilised by screen-readers and other assistive technology.
+
+## Installation
+
+`npm install #app/components/simorgh-visually-hidden-text/src`
+
+## Props
+
+| Argument  | Type | Required | Default | Example |
+| --------- | ---- | -------- | ------- | ------- |
+| No props. |      |          |         |         |
+
+## Usage
+
+```tsx
+import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
+
+const WrappingComponent = () => (
+  <VisuallyHiddenText>Some offscreen text</VisuallyHiddenText>
+);
```

Comment:
Should we also give an example of HTML within the VisuallyHiddenText component, because that's a common use case across the repo?

Reply:
Aye could do, any particularly good examples you would suggest?

Reply:
Perhaps a simple version of https://github.com/bbc/simorgh/blob/8d5d16e26d28aa78819756d85ec690cd699b2da1/src/app/pages/FrontPage/FrontPage.jsx#L78-L80 (maybe without the inner span with `lang=en-GB`?)